project/Dependencies.scala

```diff
@@ -9,7 +9,7 @@ object Dependencies {
     val scala212 = "2.12.10"
     val scala213 = "2.13.1"
 
-    val akka = "2.5.27"
+    val akka = "2.6.2"
```

Comment:
hold this until 2.6.3 since 2.6.2 is dead on arrival

Reply:
The PR is now using `2.6.4`. Is there anything we explicitly need from `2.6.4`? We'd be better off requiring _only_ `2.6.3`.  Hmm, I'll try that out.

Reply:
Ok, `2.6.4` is required because of, at least, `parasitic`.